### DISCUSSION TITLE
Keybinding shift shift

### DISCUSSION THREAD

**Discussion by peterpeterparker** [zed-industries/zed#12244] [Feature Requests]
I promise, this is my last stupid question for today 😉.

Is there a way to define a custom keybinding for clicking twice in a row on a same key, such as clicking `shift` and then `shift` again to open the search of project files?

I tried following option without success

```
{
    "bindings": {
      "shift shift": ["workspace::SendKeystrokes", "cmd-p"],
    }
}

// or

{
    "bindings": {
      "shift- shift-": ["workspace::SendKeystrokes", "cmd-p"],
    }
}

// or

{
    "bindings": {
      "shift-shift-": ["workspace::SendKeystrokes", "cmd-p"],
    }
}
```

**Comment by SomeoneToIgnore** [comment]
I believe this is not supported at all, see https://github.com/zed-industries/zed/issues/6460

**Reply by peterpeterparker** [reply]
Gotcha, thanks for the feedback!

**Reply by styx** [reply]
@peterpeterparker it is available in preview now! `"shift shift": "file_finder::Toggle"`
